### ISSUE TITLE
Unable to connect to key protected channel with /join

### ISSUE THREAD

**Issue by denisbr** [Nordaaker/convos#268]
When using `/join #channel key` to manually join a key protected channel, it always fails with:

```
irc-servername
/join #channel key: Cannot join channel (Incorrect channel key)
```

I have attempted with multiple channels that I certainly have the correct key for.

**Comment by jhthorsen** [comment]
So this bug was in both Mojo::IRC::UA and Convos::Core::Connection::IRC, so to join a channel with a key, you need to update both.

Note that I will merge this fix into the "stable" branch later.

**Comment by serhack** [comment]
I think this needs to be re-opened. It does not work, tested in Convos version 3.08.

**Comment by jhthorsen** [comment]
Should be fixed in 3.11.

**Comment by serhack** [comment]
I can confirm it works!
